### ISSUE TITLE
Gaian -2 MORALE penalty is crippling

### ISSUE THREAD

**Issue by bvanevery** [tnevolin/thinker-doer#72]
I just realized that when the Gaians build a Command Center, it does no good.  Without one, I start at Very Green.  With one, I start at Very Green.  I might have liked to have realized that, before building 4 of them.  I thought an invasion from my immediately adjacent neighbors the Caretakers was coming.  They wandered around my land for awhile, and then there were sunspots.  After that they got bored and signed a Treaty, so I was saved from the bell.

I don't see any justification for crippling the Gaians this much.  Unlike vanilla, their captured mindworms are _not_ strong weapons.  Not only do they have no offensive punch, but you've nerfed them having any real ability to level up.  +1 PLANET basically amounts to being able to make some money and get some artifacts at the beginning of the game.  It's not a military advantage anymore.

The only other things the Gaians have going for them is +2 EFFIC.  Which is pretty laughable and unimportant as faction abilities go.  It's not like you gave them the ability to print money or something.  Oh, and I guess they can eat fungus, big wow.

Finally, the Gaian faction description is wrong.  It says they get -1 MORALE -1 POLICE, but they actually get -2 MORALE.

I suggest going back to -1 POLICE and ditching any MORALE penalty.  You can still use 1 police unit when you have -1 POLICE.  It will deter the Gaians from going Democratic for awhile, as that results in -2 POLICE.  Going either Police State or Fundamentalist has always seemed in character for them.  Buncha Nazi Tree Huggers, and their high EFFIC lets them afford a Police State.

**Comment by tnevolin** [comment]
I don't remember why I modified Gaians. Will revert to vanilla in next release.

**Comment by bvanevery** [comment]
Reverting to vanilla still carries -1 POLICE -1 MORALE.  Meanwhile you have nerfed mindworm offensives.  That harms Gaian ability.  They shouldn't have onerous penalties if they don't actually have good mindworms anymore.  One or the other penalty, not both.

**Comment by tnevolin** [comment]
It's pretty hard to balance and evaluate SE choices with utmost precision. One could say if they switch to native units their +1 PLANET more than compensates -1 MORALE. The +2 EFFICIENCY gives them more credits/psych/research. That should be plenty to overweight -1 POLICE that effectively just take the nerve stapling ability.

Why Cult is better? They have extra PLANET but no efficiency, production and money flow penalty.

Oh I have no illusions how not balanced factions are. However, I prefer not to touch them leaving this to text modders. Sometime I borrow their ideas back but it should be a complete faction rebalance approach. Not just one here and there.

**Comment by bvanevery** [comment]
I beg to differ about "utmost precision" because I've done it in my mod.  I also find that the work is ongoing, due to changes elsewhere in the game I've made.  The problem is, when you change a major game system's balance, it tends to harm some specific faction's likely play style.  But since you're 1 person and you don't play the game non-stop, it can take months before you realize what you've done to some faction.  There aren't enough helpful and verbose playtesters "in the wild" for it to be otherwise.  I get useful feedback on faction balance from playtesters once in a blue moon.  All balancing had to get done by me, me, and me.

>  I prefer not to touch them leaving this to text modders. 

Nobody is going to do this for you _and_ deploy changes where others will use them.  They might twiddle with them themselves, for their own personal benefit.  Meanwhile it's _your_ work and reputation on the line, out in front of everybody.

The reason I don't drop my own factions in, in place of yours, is my factions weren't balanced for your game systems.  They were balanced for _my_ systems, _my_ SE table.  And importantly so far, my AI, and my Huge map size assumption.  Faction design isn't an orthogonal concern.  Everything depends on all the other stuff you do in _your_ mod.

> a complete faction rebalance approach. Not just one here and there.

I am never going to test, for you, _every_ single faction.  You are getting test data as it comes.  The only process for balancing factions that actually works, is to see how they perform in the real world, and then do something about what's wrong with them.  I did this for 2.5 years.  That's why mine are balanced.  There was never any "clean slate" as to how things would or should be balanced, and my considered experience at this point, is the idea is absurd.

I bet you're getting little to no test feedback from other people about your faction balance.  Players just... aren't useful, for the most part.  At least not for that.  They can find you a bug now and again.  That's useful.
